package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "discord-api-docs",
-  "version": "1.1.1",
+  "version": "1.1.2",
```

Comment:
I do not see the point of changing the version here?

Reply:
Well it was for me, but it just went with the whole thing